src/lib/api.ts

```
@@ -159,13 +161,13 @@ export const executeSettings = async (
 };
 
 export const listTemplateShop = async (): Promise<any> => {
-	throw new Error("Currently disabled as the template shop is being rethought")
+	throw new Error('Currently disabled as the template shop is being rethought');
 	const response = await axiosInstance.get(`/template-shop`);
 	return response.data;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)